javascript/ecmascript-2015/string-upgrades/unicode-in-javascript.md

```diff
@@ -30,36 +30,42 @@ links:
 
 ---
 
-# Unicode in ES6
+# Unicode in JavaScript
 
 ---
 ## Content
 
-JavaScript strings are represented using UTF-16 code units. Each code unit can be used to represent a code point in the `[U+0000, U+FFFF]` range. Code points beyond that range are represented by a surrogate pair.  
+JavaScript strings are represented using UTF-16 code units. A code unit is a 16-bit value that represents a character from the UTF-16 encoding. On top of that, each Unicode character has an ID (an integer which starts at 0) which is that character's codepoint.
 
-In ES6 the notation has been simplified:
+Each code unit can be used to represent a code point in the `[U+0000, U+FFFF]` range. Code points beyond that range are represented by a surrogate pair.  
 
-```
+Since ES6 the notation has been simplified:
+
+```js
 \\ represents U+1F332 (EVERGREEN TREE)
 "\u{1F332}"=="🌲"=="\uD83C\uDF32"
 ```
+
 As with ES5, having multiple code units per code point means that `.length` is not reliable:
```

Comment:
Maybe just remove the "As with ES5" since it doesn't add much value.

Reply:
done